src/hooks/useVisitor.ts

```diff
@@ -0,0 +1,6 @@
+const useVisitor = () => {
```

Comment:
I see that here you used an arrow function but I also saw that you used `function` for other functions in other prs. We usually prefer arrow functions since it's the newest standard when defining functions in react. That being said my main concern here is consistency it would be good to pick one of the two ways and stick with it.